src/OrchardCore.Modules/OrchardCore.Redis/HealthChecks/RedisHealthCheck.cs

```diff
@@ -35,7 +38,7 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
                 var time = await redisService.Database.PingAsync();
                 if (time > TimeSpan.FromSeconds(30))
                 {
-                    return HealthCheckResult.Unhealthy();
+                    return HealthCheckResult.Unhealthy(description: S["The Redis server isn't a live."]);
```

Comment:
"The Redis server couldn't be reached within {seconds} seconds and might be offline or have degraded performance."